paypal/paypal.lua

```diff
@@ -55,12 +55,13 @@ local inputVar = cjson.decode(io.stdin:read("*a"))
 local respbody = {}
 local reqbody = inputVar.post_data and cjson.encode(inputVar.post_data) or ''
 
+local paypalUrl = inputVar.paypal_url
```

Comment:
I've a question why we started to use _url instead of _domain? In all other connectors we used the practice to pass the domain, not the url including schema? If you don't have a strong need other than schema can be changed, let's stick to the original pattern